src/sentry/search/events/datasets/function_aliases.py

```diff
@@ -11,16 +11,19 @@
 from sentry.search.events import constants
 from sentry.search.events.types import SelectType
 from sentry.sentry_metrics.configuration import UseCaseKey
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
 
 
 def resolve_project_threshold_config(
+    # See resolve_tag_value signature
     tag_value_resolver: Callable[
-        [Optional[UseCaseKey], int, Sequence[int]], Optional[Union[int, str]]
+        [Union[UseCaseID, UseCaseKey], int, str], Optional[Union[int, str]]
     ],
-    column_name_resolver: Callable[[Optional[UseCaseKey], int, Sequence[int]], str],
+    # See resolve_tag_key signature
+    column_name_resolver: Callable[[Union[UseCaseID, UseCaseKey], int, str], str],
```

Comment:
In order to match `resolve_tag_key(use_case_id: Union[UseCaseID, UseCaseKey], org_id: int, string: str)` 

See definition: https://github.com/getsentry/sentry/blob/cf2dd1348ee1e8bf9654972705b986815306637c/src/sentry/sentry_metrics/utils.py#L141-L148